occurence.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func xenDit(i int) string {
+	// TODO: write down the logic with if statement
+	if i%3 == 0 {
+		if i%2 != 0 {
+			return "Xen"
+		}
+
+		if i%2 == 0 && i%10 != 0 {
+			return "Dit"
+		}
+
+		if i%10 == 0 {
+			return "Xendit"
+		}
+	}
+
+	return strconv.Itoa(i)
+}
```

Comment:
Based on your function, if I passed `20` what will the return?

Reply:
20